contracts/smartwallet/NativeHolderSmartWallet.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./SmartWallet.sol";
+
+/* solhint-disable no-inline-assembly */
+/* solhint-disable avoid-low-level-calls */
+
+contract NativeHolderSmartWallet is SmartWallet {
+    
+     /* solhint-disable no-unused-vars */
+    function directExecute(
+        address to,
+        bytes calldata data
+    ) external payable override returns (bool success, bytes memory ret) {
+        revert("Method not implemented");
```

Comment:
What do you think if we don't override this method here and we keep the same behaviour that the SmartWallet has?

Reply:
The only reason that I did its to avoid confusion, this contract will have two methods with the name of direct execute, I can leave both, but in one will return the native currency and in the other will not.

Reply:
As you discovered, due to the typechain version that we are using, we cannot have function overloading here, so we agreed on having 2 different functions we shouldn't have confusion, right?

Reply:
I already remove it :)